docs/Product-Strategy/Roadmap-and-Backlog.md

```diff
@@ -1,22 +1,49 @@
+Beyond that, we're still discovering and planning on what best serves our users. This doc will continue to be updated as we make decisions and scope releases. 
 
-| Functionality | Status | 
+## ATO  
+Value Delivered: Get approval for the authority to operate and create a production environment. 
+
+| Outcome | Status | 
 | -------- | ------- | 
 | User can log in using login.gov | Complete
 | Users with appropriate privileges can manage users | Complete
-| Users can upload data files by section and quarter | In Progress
+| Users can upload data files by section and quarter | Complete
 | Users with appropriate privileges can download files that were previously uploaded     | In Progress
+|Create production environment | Will happen after ATO
+
+## Release 1 (fka OFA MVP) Scope
+Value Delivered: ACF employees can use a secure method to log into the system; this would require a PIV card.
+
+Our first release to production will include the functionality built for ATO (above) and also include a secure way for ACF employees to access the system, we're currently looking into integrating with HHS AMS/ACF AMS or additional options within login.gov. It is important these measures are put into place before production data is uploaded to the system.
+
+The team is also investigating options for optimizing permission management and strategy as a part of this release. 
+| Outcome | Status | 
+| -------- | ------- | 
+| ACF employee can log in securely | Researching
+| Permissions Configuration | Researching
+
 
-## Tribal MVP Scope
-The next release will focus on enabling a subset of tribes to use the upload functionality. This will allow our application to provide value to one of our key user groups early on. We are currently in the research portion of understanding what this could look like. 
+## Release 2 (fka Tribe MVP) Scope
+
+Value Delivered: Allow users to securely upload data into our system, replacing a less secure way of doing so while increasing communication with the users and not increasing burden for OFA Admin staff.
+
+_Notes:_ We're investigating a way to automatically update file names once uploaded (based on standard naming convention) and set up a CRON job to pick up files and move them into a place where the existing TDRS application can pick them up. If this solution is selected, it would reduce the amount of time OFA Admin spends with each file. 
+
+While this type of workflow isn't our long term workflow, including it at this point delivers value to the users quicker. So we are evaluating the ROI before deciding on this path.
+
+_Risks:_ The users that would be on-boarded to this process, would potentially have to learn and adjust with future releases and changes in functionality and workflow. Increase communication with them should help here.
+| Outcome | Status | 
+| -------- | ------- | 
+| STT user of the system knows their file(s) were successfully uploaded | Not Started
```

Comment:
Hasn't this been started as part of upload work and the green alert that data has been uploaded?

Reply:
Good catch! I'll update